cumulus/parachains/runtimes/bridge-hubs/bridge-hub-rococo/src/lib.rs

```diff
@@ -203,7 +203,7 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
 	spec_name: create_runtime_str!("bridge-hub-rococo"),
 	impl_name: create_runtime_str!("bridge-hub-rococo"),
 	authoring_version: 1,
-	spec_version: 1_006_000,
+	spec_version: 1_007_000,
```

Comment:
I have no idea - whether it should be bumped here, or during release?

Reply:
Erm I'm also not sure. I wonder now if it shouldn't be a minor bump, like `1_006_001`? @bkontur? 😅

Reply:
I would not change `spec_version` here, last time for 1.6.0 tag/release, they did it in the release branch and when released they backport that to master, so I would better revert this